Feedthrough/model.c

```diff
@@ -166,9 +168,9 @@ Status setString(ModelInstance* comp, ValueReference vr, const char *const *valu
     switch (vr) {
         case vr_string:
             if (M(string) != STRING_START) {
-                freeMemory(comp, (void *)M(string));
+                free((void *)M(string));
             }
-            M(string) = duplicateString(comp, value[(*index)++]);
+            M(string) = strdup(value[(*index)++]);

```

Comment:
strdup neither is C89, C99 nor C11, but was only recently added to the next C standard.